bigchaindb/upsert_validator/validator_election.py

```diff
@@ -61,3 +61,6 @@ def on_approval(self, bigchain, new_height):
         # TODO change to `new_height + 2` when upgrading to Tendermint 0.24.0.
         bigchain.store_validator_set(new_height + 1, updated_validator_set)
         return encode_validator(self.asset['data'])
+
+    def on_rollback(self, bigchaindb, new_height):
+        bigchaindb.delete_validator_set(new_height)
```

Comment:
`new_height` is the pre-commit height, so if an election is concluded at `new_height` the validator set will be stored at `new_height + 1`